core/src/main/resources/META-INF/resources/primefaces-extensions/timer/timer.js

```diff
@@ -49,7 +50,7 @@ PrimeFaces.widget.ExtTimer = PrimeFaces.widget.BaseWidget.extend({
             if ("percentage" === format) {
                 value = ((this.currentTimeout * 100) / this.originalTimeout) + "%";
             } else if ("human" === format) {
-                value = moment.duration(this.currentTimeout, 'seconds').humanize();
+                value = moment.duration(this.currentTimeout, 'seconds').locale(this.locale).humanize();
             } else {
                 value = moment.utc(moment.duration(this.currentTimeout, 'seconds').asMilliseconds()).format(format);
```

Comment:
Should this moment value here also use locale?

Reply:
Right, I will add the same for custom format